Student_BCOs/June_Beshea_BCO_000550.json

```diff
@@ -0,0 +1,242 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000550",
+    "checksum": "5f27243107121447b7f2a33b1cd7ac91f2444714db553cf739750600e82ba60e",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Correlating miRNA and gene expression data in 34 cancer phenotypes",
+        "version": "1.0.0",
+        "license": "Creative Commons Attribution 4.0 International License",
+        "created": "2019-10-17T02:13:35.166914",
+        "modified": "2019-10-17T21:22:33.269689",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": " Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-16"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "June Beshea",
+                "orcid": "https://orcid.org/0000-0001-9101-8000",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "jbeshea@gwu.edu"
+            },
+            {
+                "name": "Mahmoud Ahmed",
+                "orcid": "https://orcid.org/0000-0002-4377-6541",
+                "affiliation": "Gyeongsang National University",
+                "contribution": [],
+                "email": ""
```

Comment:
even though you don't have the ORCIDS of all the other authors, you still need to list ALL the authors and their contribution type #113